op-node/eth/sync_status.go

```diff
@@ -35,4 +35,6 @@ type SyncStatus struct {
 	// UnsafeL2SyncTarget points to the first unprocessed unsafe L2 block.
 	// It may be zeroed if there is no targeted block.
 	UnsafeL2SyncTarget L2BlockRef `json:"queued_unsafe_l2"`
+	// HightestIndex is the highest index of op-node. only used in op-batcher and op-proposer for HA.
+	HightestIndex int64 `json:"hightest_index"`
```

Comment:
The "highest index" is an internal concept that only applies to the external coordinator mode and should not be exposed externally. And exposing this new field breaks API compatibility.
We should maintain this index internally only.

Reply:
if this changes only in HA branch, I think it's OK. 
and these functions are likely already in the op-coordinator, I think these should be implemented in op-coordinator